util/topgen.py

```diff
@@ -955,6 +962,18 @@ def main():
     log.info("Detected crossbars: %s" %
              (", ".join([x["name"] for x in xbar_objs])))
 
+    # If specified, override the seed for random netlist constant computation.
+    if args.seed is not None:
```

Comment:
does `if args.seed:` directly work here?

Reply:
yes that works, thanks!